cadctl/cmd/credentials/credentials.go

```diff
@@ -0,0 +1,73 @@
+package credentials
```

Comment:
**suggestion:** I would probably rename the subfolder of this package to something like `periodic-checks`. Maybe you find a better term, the code can be used as non-periodic check, but I want to clearly separate it from all the stuff we have going on with PD routing.